app/src/main/java/de/tum/in/tumcampusapp/adapters/SimpleStickyListHeadersAdapter.java

```diff
@@ -0,0 +1,87 @@
+package de.tum.in.tumcampusapp.adapters;
+
+import android.content.Context;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.BaseAdapter;
+import android.widget.TextView;
+
+import java.util.List;
+
+import de.tum.in.tumcampusapp.R;
+import se.emilsjolander.stickylistheaders.StickyListHeadersAdapter;
+
+abstract public class SimpleStickyListHeadersAdapter<T extends SimpleStickyListHeadersAdapter.SimpleStickyListItem>
```

Comment:
The reason why I create this class is that I need to use to different but similar StickyHeader Adapter. This works as a subclass for the two adapters, so I don't need to implement methods which are exactly the same in two classes.